litgpt/model.py

```diff
@@ -301,8 +301,8 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
 class Phi3MLP(nn.Module):
     def __init__(self, config: Config) -> None:
         super().__init__()
-        self.gate_up_proj = nn.Linear(config.n_embd, config.intermediate_size, bias=config.bias)
-        self.down_proj = nn.Linear(config.intermediate_size//2, config.n_embd, bias=config.bias)
+        self.gate_up_proj = nn.Linear(config.n_embd, 2 * config.intermediate_size, bias=config.bias)
```

Comment:
Huh, that's what I had originally, because it matches their description, but I got dimension mismatches with that.